### PR TITLE
[CSLD-65] PVSS certificates clarification

### PR DESCRIPTION
**Pull request by denisshevchenko**

### CODE REVIEW

File: _docs/2017-01-19-glossary.md [hunk 1]
```diff
@@ -127,6 +127,10 @@ owns. By this percentage we mean "all the value held by wallets on the particula
 node divided by the total value in the system". This relative value is called
 _stake_, and the node chosen to produce a new block is called a [slot leader](#slot-leader).
 
+## Richmen
```
Comment (rockbmb) [inline _docs/2017-01-19-glossary.md]: Richmen -> Richman

**Review by rockbmb — APPROVED** [review]

File: _docs/technical/2017-02-10-pvss.md [hunk 2]
```diff
@@ -63,3 +63,18 @@ Since decrypted share contains a proof, we can be sure that decrypted share is t
...
+2. Piblic key used for signing (e.g. [signing key](https://github.com/input-output-hk/cardano-sl/blob/c38bcba3f16bf5dd12fa4842707d1dfd360c0f65/src/Pos/Ssc/GodTossing/Core/Types.hs#L153)).
+3. Index of an expiry epoch (e.g. the last epoch when this certificate is valid).
+4. Signature of a pair `(A,B)`, where `A` is a VSS key and `B` is an epoch index.
+
+Initially, all stakeholders with stake enough for participation in randomness generation (we call them [richmen](/glossary/#richmen)) [have their own certificates](https://github.com/input-output-hk/cardano-sl/blob/281aa4e75d6de5d13b00b1e40f4e3dbdc0dd11fe/src/Pos/Ssc/GodTossing/Workers.hs#L123). When a new stakeholder with enough stake appears or when an existing certificate expires, a new certificate should be generated and [submitted to the network](https://github.com/input-output-hk/cardano-sl/blob/281aa4e75d6de5d13b00b1e40f4e3dbdc0dd11fe/src/Pos/Ssc/GodTossing/Workers.hs#L125). Other nodes accept this certificate if it's valid and if node has enough stake. Certificates are stored in blocks.
```
Comment (rockbmb) [inline _docs/technical/2017-02-10-pvss.md]: "with stake enough"
maybe you wanted "with enough stake"?

**Review by rockbmb — CHANGES_REQUESTED** [review]
@staenrey can you review this please?

**Review by staenrey — APPROVED** [review]
Changes made

**Review by rockbmb — APPROVED** [review]

**Pull request merged**